src/JustEat.StatsD/IStatsDTransport.cs

```diff
@@ -1,7 +1,9 @@
+using System;
+
 namespace JustEat.StatsD
 {
     public interface IStatsDTransport
     {
-        void Send(string metric);
+        void Send(ReadOnlySpan<byte> metric);
```

Comment:
Would be a breaking change and prevent use by any TFM that isn't 2.1+ compatible.